pages/index.html

```diff
@@ -6,22 +6,22 @@
 ---
 <div class="header"><em>"Help us preserve the past!"</em></div><br>
 <p>
-    Omniarchive is a community that focuses on the preservation of all official Minecraft content. Although we archive
-    all editions of Minecraft, our main focus is finding and archiving versions of Java Edition that are lost to time
+    Omniarchive is a community that focuses on preserving all official Minecraft content. Although we archive all 
+    editions of Minecraft, our main focus is finding and archiving versions of Java Edition that are lost to time
     and are no longer available.<br><br>
 
     Since its inception in September 2017 by HalfOfAKebab, the Omniarchive community has recovered more than 80
     previously missing versions, mostly from the game's early development stages during 2009-2011. We are actively
     looking for more versions, with our usual methods being to contact players that played during these early days.<br><br>
 
-    Our community also actively focuses on documentation of the early history of Minecraft and its community, so it can
-    be preserved for all to see. This work includes finding proof of existence of versions previously unknown to exist,
-    archiving screenshots and videos, and contributing to the Minecraft Wiki.<br><br>
+    Our community also actively focuses on documenting the early history of Minecraft and its community, so it can
+    be preserved for all to see. This work includes finding proof of the existence of versions previously unknown 
```

Comment:
partial to "existence proof" instead of "proof of existence" compared to "proof of the existence"

Reply:
> Our community also actively focuses on documenting the early history of Minecraft and its community, so it can be preserved for all to see. This work includes finding **existence proof** of versions previously unknown to exist, archiving screenshots and videos, and contributing to the Minecraft Wiki.

I think "proof of the existence of versions" sounds better than "existence proof of versions". Any reason why you'd prefer that?

Reply:
Let's ignore my midnight brain and push this!